src/main/scala/io/iohk/ethereum/merklepatriciatrie/Node.scala

```diff
@@ -0,0 +1,98 @@
+package io.iohk.ethereum.merklepatriciatrie
```

Comment:
Maybe the package could renamed to `mpt`? Especially that this name is referred to in so many places.